File: JS-Basics/Conditional Statements Advanced/Exercise/05.js
function vacationBudget(budget, season) {
    let destination = '';
    let place = '';
    let moneySpend = 0;

    if (budget <= 100) {
        destination = "Bulgaria";
      
        moneySpend = season === "summer" 
          ? budget * 0.3 : budget * 0.7;
      
        place = season === "summer" 
          ? "Camp" : "Hotel";
      
    } else if (budget <= 1000) {
        destination = "Balkans";
      
        moneySpend = season === "summer" 
          ? budget * 0.4 : budget * 0.8;
      
        place = season === "summer" 
          ? "Camp" : "Hotel";
      
    } else if (budget > 1000) {
        destination = "Europe";
        moneySpend = budget * 0.9;
        place = "Hotel";
    }

    console.log(`Somewhere in ${destination}`);
    console.log(`${place} - ${moneySpend.toFixed(2)}`);
}

File: Basics/Conditional Statements Advanced/Exercise/05.py
budget = float(input())
season = input()

destination = ''
place = ''
money_spend = 0

if budget <= 100:
    destination = "Bulgaria"
  
    if season == "summer":
        money_spend = budget * 0.3
        place = "Camp"
    elif season == "winter":
        money_spend = budget * 0.7
        place = "Hotel"
      
elif budget <= 1000:
    destination = "Balkans"
  
    if season == "summer":
        money_spend = budget * 0.4
        place = "Camp"
    elif season == "winter":
        money_spend = budget * 0.8
        place = "Hotel"
      
elif budget > 1000:
    destination = "Europe"
    place = "Hotel"
    money_spend = budget * 0.9

print(f"Somewhere in {destination}")
print(f"{place} - {money_spend:.2f}")
